### PR TITLE
outline for controller and presenter

### PR DESCRIPTION
**Pull request by rhklee**

### CODE REVIEW

File: arbeitszeit_web/www/presenters/request_user_password_reset_presenter.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases import request_user_password_reset as use_case
+
+class RequestUserPasswordResetViewModel:
+    # @TODO: Does this need to be implemented?
```
Comment (Amittai-Aviram) [inline arbeitszeit_web/www/presenters/request_user_password_reset_presenter.py:6]: Is this class supposed to create a page that confirms that the user has successfully changed his or her password?  If the answer to your TODO question is "no," how does the user get confirmation that the password change has succeeded?

File: arbeitszeit_web/www/presenters/request_user_password_reset_presenter.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    pass
+
+@dataclass
+class RequestUserPasswordResetPresenter:
+
+    def render_response(self, response: use_case.Response) -> RequestUserPasswordResetViewModel:
+
+        raise NotImplemented("not impled")
```
Comment (Amittai-Aviram) [inline arbeitszeit_web/www/presenters/request_user_password_reset_presenter.py:14]: Let's be consistent and spell it out:  "Not implemented."

File: arbeitszeit_web/www/controllers/request_user_password_reset_controller.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+from arbeitszeit.repositories import DatabaseGateway
+
+
+@dataclass
+class RequestUserPasswordResetController:
+    database: DatabaseGateway
+
+    def process_user_password_reset_request(self):
+        raise NotImplemented("not impled")
```
Comment (Amittai-Aviram) [inline arbeitszeit_web/www/controllers/request_user_password_reset_controller.py:11]: "Not implemented."  Actually, does the exception itself print a default message?  If so, you don't need a string at all.

**Review by Amittai-Aviram — COMMENTED** [review]
I left just a couple of questions and comments.

**Comment by 4lm** [comment]
Question. Was this PR made obsolete by #1043 and can be closed?

**Comment by sloschert** [comment]
@4lm No, it is a different issue, this one deals with changing the password, not the email address (see open feature request https://github.com/ida-arbeitszeit/arbeitszeitapp/issues/652). I hope that @rhklee will find time to keep working on it.

**Pull request closed without merging**